ynput/ayon-core
# Chore: Move hiero client code

## Changelog Description
Moved Hiero client code next to server codebase.

## Additional information
Removed settings of not available plugins `ExtractReviewCutUpVideo` and `CollectInstanceVersion`.
This PR is also adding fixes from https://github.com/ynput/ayon-core/pull/589

## Testing notes:
1. Create package of hiero and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Hiero integration should work as before.
